modules/virt-creating-servicemonitor-resource-for-node-exporter.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-servicemonitor-resource-for-node-exporter_{context}"]
+= Creating a ServiceMonitor resource for the node-exporter service
+
+To use the metrics exposed by the `node-exporter service`, you must configure {product-title} Container Platform monitoring to scrape metrics from the `/metrics` endpoint. You can do this by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the `node-exporter` service should be monitored.
```

Comment:
the rendered output shows "Container Platform" twice:
"...you must configure OpenShift Container Platform Container Platform..."